persistence/mongodb-record/src/main/scala/net/liftweb/mongodb/record/field/ObjectIdField.scala

```diff
@@ -58,39 +46,60 @@ class ObjectIdField[OwnerType <: BsonRecord[OwnerType]](rec: OwnerType)
   }
 
   def setFromJValue(jvalue: JValue): Box[ObjectId] = jvalue match {
-    case JNothing|JNull if optional_? => setBox(Empty)
+    case JNothing | JNull if optional_? => setBox(Empty)
     case JObject(JField("$oid", JString(s)) :: Nil) => setFromString(s)
     case JString(s) => setFromString(s)
     case other => setBox(FieldHelpers.expectedA("JObject", other))
   }
 
-  def setFromString(in: String): Box[ObjectId] =
-    if (ObjectId.isValid(in))
-      setBox(Full(new ObjectId(in)))
-    else
-      setBox(Failure("Invalid ObjectId string: "+in))
+  def setFromString(in: String): Box[ObjectId] = {
+    if (ObjectId.isValid(in)) setBox(Full(new ObjectId(in)))
+    else setBox(Failure(s"Invalid ObjectId string: $in"))
```

Comment:
I'm really not a fan of these compacted if statements. Can we please break this into a multi-line if statement with brackets around the "then" parts? (Unlike the previous version which lacked those brackets.